dessia_common/schemas/core.py

```diff
@@ -940,6 +954,17 @@ def classes_are_standalone_consistent(self) -> PassedCheck:
               f"are not consistent. They should be all standalone in db or none of them should."
         return WrongType(msg)
 
+    def get_import_names(self, import_names):
+        """ Process UnionProperty and get/update the import names list."""
+        import_name = f"{self.origin.__module__}.{self.origin._name}"
```

Comment:
As we discussed yesterday, this could be set as a cached_property in TypingProperty class

Reply:
We can't set this as a cached_property because I need to give import_names list

Reply:
it's done as Property